icons/trending-up-down.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Alportan"
+  ],
+  "tags": [
+    "arrow",
+    "arrows",
+    "estimated"
+    "up"
+    "down"
+  ],
+  "categories": [
+    "arrows"
```

`trending-up/down` are in `charts`, let's also add this one.

```suggestion
    "charts",
    "arrows"
```